ProteoformSuiteInternal/Proteoform.cs

```diff
@@ -39,7 +39,7 @@ public Proteoform(string accession)
 
         public List<Proteoform> get_connected_proteoforms()
         {
-            return relationships.Where(r => r.peak.peak_accepted).SelectMany(r => r.connected_proteoforms).ToList();
+            return relationships.Where(r => r.accepted).SelectMany(r => r.connected_proteoforms).ToList();
```

Comment:
These relation/peak accepted things are confusing. Let's maybe think of a way to clarify or streamline those.